java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/WindowAggPlan.scala

```diff
@@ -174,9 +180,19 @@ object WindowAggPlan {
       distributionDf
     }
 
+    var approxMinCount = Long.MaxValue
+    val rows = distributionDf.select(distinctCountColName).collect()
+    for (row <- rows) {
+      val count = row.getLong(0)
+      approxMinCount = math.min(approxMinCount, count)
+    }
+    val minCount = approxMinCount / (1 + approxRatio)
```

Comment:
the type of minCount is float, we may use `math.ceil` to get the distinct count  in `int`